### ISSUE TITLE
macOS build fails with error cannot find GeneratedPluginRegistrant.swift:12:3

### ISSUE THREAD

**Issue by ptheofan** [mogol/flutter_secure_storage#486]
```
Building macOS application...
/Users/xxx/Sites/insta_sensors/macos/Flutter/GeneratedPluginRegistrant.swift:12:3: error: cannot find 'FlutterSecureStorageMacosPlugin' in scope
  FlutterSecureStorageMacosPlugin.register(with: registry.registrar(forPlugin: "FlutterSecureStorageMacosPlugin"))
  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Am I doing something wrong or is it an issue with the latest XCode?

**Comment by valientegaston** [comment]
Same issue here

**Comment by Megamil** [comment]
I know it's not correct, but just to help temporarily, but just change from FlutterSecureStorageMacosPlugin to SwiftFlutterSecureStoragePlugin in the generated file in Flutter/GeneratedPluginRegistrant

**Comment by felipecastrosales** [comment]
Same problem here. :(

**Comment by juliansteenbakker** [comment]
Which version of flutter_secure_storage are you using?

**Comment by ptheofan** [comment]
Using the latest stable, version 7.0.0
These are the details in the .lock
```
  flutter_secure_storage:
    dependency: "direct main"
    description:
      name: flutter_secure_storage
      url: "https://pub.dartlang.org"
    source: hosted
    version: "7.0.0"
  flutter_secure_storage_linux:
    dependency: transitive
    description:
      name: flutter_secure_storage_linux
      url: "https://pub.dartlang.org"
    source: hosted
    version: "1.1.2"
  flutter_secure_storage_macos:
    dependency: transitive
    description:
      name: flutter_secure_storage_macos
      url: "https://pub.dartlang.org"
    source: hosted
    version: "2.0.0"
  flutter_secure_storage_platform_interface:
    dependency: transitive
    description:
      name: flutter_secure_storage_platform_interface
      url: "https://pub.dartlang.org"
    source: hosted
    version: "1.0.1"
  flutter_secure_storage_web:
    dependency: transitive
    description:
      name: flutter_secure_storage_web
      url: "https://pub.dartlang.org"
    source: hosted
    version: "1.1.1"
  flutter_secure_storage_windows:
    dependency: transitive
    description:
      name: flutter_secure_storage_windows
      url: "https://pub.dartlang.org"
    source: hosted
    version: "1.1.3"
```

**Comment by valientegaston** [comment]
> I know it's not correct, but just to help temporarily, but just change from FlutterSecureStorageMacosPlugin to SwiftFlutterSecureStoragePlugin in the generated file in Flutter/GeneratedPluginRegistrant

It doesn't work for me. GeneratedPluginRegistrant.swift regenerate itself on build action and build fails.  :(

Launching lib/main.dart on macOS in debug mode...
lib/main.dart:1
/Users/gaston/Desktop/Proyecto_llaves/desktop/llavero/macos/Flutter/GeneratedPluginRegistrant.swift:11:3: error: cannot find 'FlutterSecureStorageMacosPlugin' in scope
  FlutterSecureStorageMacosPlugin.register(with: registry.registrar(forPlugin: "FlutterSecureStorageMacosPlugin"))
  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
** BUILD FAILED **

Exception: Build process failed
Exited

**Comment by Shahriyar13** [comment]
Building macOS application...
[project]/macos/Flutter/GeneratedPluginRegistrant.swift:14:3: error: cannot find 'FlutterSecureStorageMacosPlugin' in scope
  FlutterSecureStorageMacosPlugin.register(with: registry.registrar(forPlugin: "FlutterSecureStorageMacosPlugin"))

**Comment by Shahriyar13** [comment]
please update pub.dev

**Comment by anaerobic** [comment]
> please update pub.dev

@juliansteenbakker 🙏🏻

**Comment by krsatz** [comment]
Is this fixed? Still facing the same issue.

**Comment by juliansteenbakker** [comment]
I will update pub.dev after christmas, sorry for the delay

**Comment by juliansteenbakker** [comment]
I have released a new version 2.0.1 of the macos implementation. I think running pub upgrade will upgrade the dependency automatically for you.

**Comment by Rohithgilla12** [comment]
If someone stumbles upon this, you can try this.
```yaml
dependency_overrides:
  flutter_secure_storage_macos: ^2.0.1
```
